### PR TITLE
Window on bottom (configurable)

### PR DESCRIPTION
**Pull request by dubiousdavid**

### CODE REVIEW

File: fzf.el [hunk 1]
```diff
@@ -44,7 +44,7 @@
   "Configuration options for fzf.el"
   :group 'convenience)
 
-(defcustom fzf/window-height 15
+(defcustom fzf/window-height 10
```
Comment (bling) [inline fzf.el]: can you keep this at 15?  also, if you can squash everything to a single commit that'd be great.  thanks.

**Comment by denisinla** [comment]
+1 on this.

File: fzf.el (hunk 1 above)
Comment (jkrmr) [inline fzf.el]: @dubiousdavid

**Comment by dubiousdavid** [comment]
Changed the default window height to 15.

**Comment by dubiousdavid** [comment]
@bling You can squash merge the PR btw. See: https://github.com/blog/2141-squash-your-commits

**Pull request merged**